modules/docopt.py

```diff
@@ -1,66 +0,0 @@
-"""
-Usage:
-  RoomMgt
-  RoomMgt create_room <room_type> <room_name> ...
-
```

Comment:
This file was changed to main.py becuase the docopt.py file name causes import errors. the main.py file has the complete functionality